### PR TITLE
[15.0][MIG] project_hr: Migration to 15.0

### PR DESCRIPTION
**Pull request by chandni299**

### CODE REVIEW

**Comment by chandni299** [comment]
@pedrobaeza @sbidoul , how we can solve this pre-commit error?
[https://github.com/OCA/project/actions/runs/4066569768/jobs/7002827817](https://github.com/OCA/project/actions/runs/4066569768/jobs/7002827817)

**Comment by sbidoul** [comment]
@chandni299 run `pipx run copier -f update` in the repo, and commit the resulting change.

**Comment by pedrobaeza** [comment]
I have done it for you on #1059

**Comment by chandni299** [comment]
It works :)
Thank you so much for your help here @pedrobaeza @sbidoul :+1:

**Review by pilarvargas-tecnativa — APPROVED** [review]
LGTM :+1:

File: project_hr/models/project_task.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+# Copyright 2019 Brainbean Apps (https://brainbeanapps.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, exceptions, fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    employee_id = fields.Many2one(
```
Comment (pedrobaeza) [inline project_hr/models/project_task.py]: This field should be changed as well as the `user_ids` to many2many `employee_ids`.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by pilarvargas-tecnativa** [comment]
@chandni299 Can you make the changes Pedro suggested?

File: project_hr/models/project_task.py (hunk 1 above)
Comment (chandni299) [reply project_hr/models/project_task.py]: Hello @pedrobaeza , Thank you for your review.
I have added the requested changes. You can have a look.

**Comment by chandni299** [comment]
> @chandni299 Can you make the changes Pedro suggested?

Sure. Done

**Comment by pedrobaeza** [comment]
Thanks for the changes, @chandni299. Please squash the latest commit with the migration one, as the change to m2m is part of the migration itself.

**Comment by pedrobaeza** [comment]
Thanks for the changes, @chandni299. Please squash the latest commit with the migration one, as the change to m2m is part of the migration itself.

**Comment by chandni299** [comment]
The latest commit is squashed with the migration one @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]
Thanks

/ocabot merge nobump

**Pull request merged**